appeals/web/src/server/appeals/appeal-details/start-case/__tests__/__snapshots__/start-case.test.js.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`start-case GET /start-case/date should render "Case started" confirmation page should 1`] = `
```

Comment:
Is the route/path correct here? I'd expect the path to be something like `/confirmation` for a confirmation screen. Also there seems to be a typo (extra `should` at the end)

Reply:
done